uber/models.py

```diff
@@ -1654,6 +1654,14 @@ class MPointsForCash(MagModel):
     when        = Column(UTCDateTime, default=lambda: datetime.now(UTC))
 
 
+class PromoCode(MagModel):
+    price           = Column(Integer, default=0)
+    uses            = Column(Integer, default=1)
+    code            = Column(UnicodeText)
+    users           = Column(UnicodeText, default="")
```

Comment:
Dom has informed me to make a second class called PromoCodeUsages which has a promo_id and an attendee_id as foreign keys to link them